settings.gradle

```diff
@@ -2,6 +2,5 @@ include ':AndroidSDKCore'
 include ':AndroidSDKLocation'
 include ':AndroidSDKPush'
```

Comment:
are we thinking of combining leanplum-push and leanplum-fcm in a future PR ?

Reply:
Yes. In future for sure or in our new architecture . I tried doing it. But, there is lot more changes than i expected. So, not for this commit.